pkgs/development/interpreters/python/build-python-package-pyproject.nix

```diff
@@ -5,24 +5,27 @@
 }:
 
 {
+# passed to "python setup.py"
+  setupPyDistFlags ? []
 # passed to "python setup.py build_ext"
 # https://github.com/pypa/pip/issues/881
 # Rename to `buildOptions` because it is not setuptools specific?
-  setupPyBuildFlags ? []
+,  setupPyBuildFlags ? []
 # Execute before shell hook
 , preShellHook ? ""
 # Execute after shell hook
 , postShellHook ? ""
 , ... } @ attrs:
 
 let
+  installOptions = lib.concatMapStringsSep " " (option: "--install-option ${option}") setupPyDistFlags;
   options = lib.concatMapStringsSep " " (option: "--global-option ${option}") setupPyBuildFlags;
```

Comment:
Also if we rename `setupPyBuildFlags` option to pass global flags instead we also need to add them to `options` here as `--global-option`s instead of `--install-options`.

Reply:
Speaking of which, is that right to use `--global-option` instead of some kind of `--build-option` here for `setupPyBuildFlags`?